src/main/scala/com/gu/riffraff/artifact/RiffRaffArtifact.scala

```diff
@@ -188,6 +199,22 @@ object RiffRaffArtifact extends AutoPlugin {
 
       riffRaffUpload := (riffRaffUpload dependsOn (test in Test)).value
     )
+
+    def staticPackage(packageDirectory: File): Seq[(File, String)] = {
+      def listRegularFiles(file: File): Seq[File] = {
```

Comment:
Have you looked at the nio `Files` API for this? I was pleasantly surprised by how easily it accomplished this type of thing:

https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html and in particular https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#walk-java.nio.file.Path-java.nio.file.FileVisitOption...-